css/styles.css

```diff
@@ -0,0 +1,113 @@
+.DefaultProgressBar__vertical {
```

Comment:
should this file be gitignored (altho not npm-ignored), and generated as part of build output?

Reply:
Hmm what is the rationale for not putting it on Github? On one hand it's a generated file, but on the other it might be useful for someone skimming the repository online or right after cloning.

Reply:
generally those aren't supported use cases; you can use http://unpkg.com/react-dates/ if you want to browse generated output.